FlexConfirmMail/Outlook
# Improve design of dialog to confirm new domain

## Overview

Improve the design of the dialog to confirm new domain.

The message of reconfirmation dialogs are currently very flat and it is not clear what we should check about.
With this change, embedded addresses, domains, and other important part is emphasized more.

## Changes

### Bofore

Having used the default dialog.

![image](https://github.com/FlexConfirmMail/Outlook/assets/15982708/874ca47f-da07-4ad9-9b1c-e0c5e0d85aa4)

### After

Using a newly created original dialog for confirmation.

![image](https://github.com/FlexConfirmMail/Outlook/assets/15982708/589fc9ab-4357-4a1e-b8fa-574049d17b8a)

## How to verify the fixed issue

1. Specify "Confirm when any recipients with domains different from any existing recipients are added" from setting dialog.
2. Reply a mail.
3. Add some new domain addresses to `To` , `Cc` or `Bcc` .
4. Send mail.
